Call vim with -u NONE instead of -u /dev/null
This prevents plugins from being loaded, and fixes
http://bugs.gentoo.org/show_bug.cgi?id=306569

---
--- gtkdoc-fixxref.in
+++ gtkdoc-fixxref.in
@@ -455,7 +455,7 @@
     close (NEWFILE);
 
     # format source
-    system "echo 'let html_number_lines=0|let html_use_css=1|let use_xhtml=1|syn on|e $temp_source_file|run! syntax/2html.vim|wa!|qa!' | @HIGHLIGHT@ -n -e -u /dev/null -T xterm >/dev/null";
+    system "echo 'let html_number_lines=0|let html_use_css=1|let use_xhtml=1|syn on|e $temp_source_file|run! syntax/2html.vim|wa!|qa!' | @HIGHLIGHT@ -n -e -u NONE -T xterm >/dev/null";
 
     my $highlighted_source;
     {
